Scheduled GitHub Action "UpdateDockerImages" fails

Scheduled GitHub Action "[UpdateDockerImages](https://github.com/exasol/r-exasol/actions/workflows/update_docker_images.yml)" fails in job [Push docker image for R=bionic-cran35/3.5.3-1bionic](https://github.com/exasol/r-exasol/actions/runs/5481365729/jobs/9985584107#logs) with the following error:

```
Step 18/24 : RUN  Rscript -e 'devtools::install_github("jimhester/covr")'
 ---> Running in 25b1ab777647
Error in loadNamespace(name) : there is no package called ‘devtools’
Calls: :: ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted
The command '/bin/sh -c Rscript -e 'devtools::install_github("jimhester/covr")'' returned a non-zero code: 1

Error: Process completed with exit code 1.
```

--- a/.github/workflows/update_docker_images.yml
+++ b/.github/workflows/update_docker_images.yml
@@ -3,6 +3,7 @@ name: UpdateDockerImages
 on:
   schedule:
     - cron:  '0 0 * * *'
+  push:
 jobs:
   prep-env:
     runs-on: ubuntu-20.04

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@ Leading the TPC-H benchmark, it is considered the fastest analytical data wareho
 
 ## Status
 
-The package gets tested in Github against 4.0, 4.1, 4.2 and 4.3. You can use the [remotes package](https://remotes.r-lib.org/) to install specific versions of RODBC. The packages DBI 0.3.1 and a fork of DBItest 1.0-1 are not the current versions. The Exasol R package may however work with DBI 0.4.1, but until this is resolved the fitting older packages can be loaded from here:
+Github CI build validates the package r-exasol against R versions 4.0, 4.1, 4.2, and 4.3. You can use the [remotes package](https://remotes.r-lib.org/) to install specific versions of RODBC. Please note that packages DBI 0.3.1 and a fork of DBItest 1.0-1 are not the latest versions. The Exasol R package may however work with DBI 0.4.1, but until this is resolved appropriate and compatible older packages can be loaded from here:
 
 - https://github.com/marcelboldt/DBI
 - https://github.com/marcelboldt/DBItest

--- a/README.md
+++ b/README.md
@@ -20,7 +20,7 @@ Leading the TPC-H benchmark, it is considered the fastest analytical data wareho
 
 ## Status
 
-The package gets tested in Github against 4.0, 4.1 and 4.2 (and 3.5/3.6 for Linux!). For R 3.6 and earlier, an older RODBC package is necessary, the newer versions (1.3-17 and upwards) only support R 4.0. You can use the [remotes package](https://remotes.r-lib.org/) to install specific versions of RODBC. The packages DBI 0.3.1 and a fork of DBItest 1.0-1 are not the current versions. The Exasol R package may however work with DBI 0.4.1, but until this is resolved the fitting older packages can be loaded from here:
+The package gets tested in Github against 4.0, 4.1, 4.2 and 4.3. You can use the [remotes package](https://remotes.r-lib.org/) to install specific versions of RODBC. The packages DBI 0.3.1 and a fork of DBItest 1.0-1 are not the current versions. The Exasol R package may however work with DBI 0.4.1, but until this is resolved the fitting older packages can be loaded from here:
 
 - https://github.com/marcelboldt/DBI
 - https://github.com/marcelboldt/DBItest

--- a/doc/changes/changes_7.2.0.md
+++ b/doc/changes/changes_7.2.0.md
@@ -16,6 +16,9 @@ t.b.d.
  - #139: Updated ODBC driver source for integration test
  - #138: Fixed bug in exa.write
  - #143: Fixed Github actions
+ - #146: Update cran repo used in CI to focal-cran40 and with that removes tests for Ubuntu 18.04 and R3.5/6, because they are end of life.
+
+
 
 ## Documentation
 

--- a/tests/Dockerfile
+++ b/tests/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
 
 ARG DEBIAN_FRONTEND=noninteractive

--- a/tests/cran-submission/Dockerfile
+++ b/tests/cran-submission/Dockerfile
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
 
 ARG DEBIAN_FRONTEND=noninteractive

--- a/tests/scripts/cran/get_cran_repos.sh
+++ b/tests/scripts/cran/get_cran_repos.sh
@@ -3,5 +3,5 @@
 set -euo pipefail
 
 #hard coded list of cran repositories, from which we pick all major releases
-supported_cran_repos=("bionic-cran40" "bionic-cran35")
+supported_cran_repos=("focal-cran40")
 echo "${supported_cran_repos[@]}"

